src/definition/persistence/IPersistenceItem.ts

```diff
@@ -0,0 +1,7 @@
+import { RocketChatAssociationRecord } from '../../definition/metadata';
+
+export interface IPersistenceItem {
+    appId: string;
+    data: { [key: string]: object };
+    associations?: Array<RocketChatAssociationRecord>;
```

Comment:
It seems weird to me for the `associations` field to be optional 👀 can you please confirm?

Reply:
Yes, it should be optional to conform to our persistence model. We have methods to CRUD an item without the `associations` field ([example](https://github.com/RocketChat/Rocket.Chat/blob/f2209eecda95b7a59f04b73e14a79cae9be2dd05/apps/meteor/app/apps/server/bridges/persistence.ts#L18)).

Reply:
That method is definitely weird, but let's leave this investigation for a better moment